package.json

```diff
@@ -52,6 +55,8 @@
     "ember-cli-stylelint": "^1.0.0",
     "ember-cli-template-lint": "^0.7.5",
     "ember-cli-test-loader": "^1.1.0",
+    "ember-cli-tslint": "^0.1.3",
+    "ember-cli-typescript": "^1.0.6",
```

Comment:
Seems like we want `1.1.x`, given the changes we'll likely depend on from 1.1.

You've got 1.1 in the lockfile, though, so that doesn't really matter.

Reply:
Ya lockfile is the source of truth. Running yarn upgrade only modifies the lockfile. I updated to the latest bugfix version just now.